### PR TITLE
Add storybook and stories

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | First integration of storybook on develop
| Type?         | new feature
| BC breaks?    | no
| Deprecations? | no
| How to test?  | You can test it here : https://neomaking.github.io/prestashop-ui-kit/storybook-static/, or clone the repo, `npm i && npm run storybook`

### CODE REVIEW

**Comment by eternoendless** [comment]
Don't you think this should be in an independent repo?

**Comment by NeOMakinG** [comment]
> Don't you think this should be in an independent repo?

Why ?

**Comment by NeOMakinG** [comment]
I think we shouldn't see it as a different project, but as an addons of the UIKit

**Comment by NeOMakinG** [comment]
@eternoendless I updated the PR, It's a complete refacto, including Chromatic github action, markup separated from stories logic, adding color blindness plugin, accessibility plugin, doc plugin and building it to serve on https://neomaking.github.io/prestashop-ui-kit/storybook-static/

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,4 @@ node_modules/*
 .projectile
 .tern-port
 TAGS
+.DS_Store
```
Comment (atomiix) [inline .gitignore:13]: I don't think OS-related files should be added in the .gitignore 🤔

**Review by atomiix — COMMENTED** [review]
Are *bundle.js files versionned on purpose?

File: .gitignore (hunk 1 above)
Comment (NeOMakinG) [reply .gitignore:13]: You're probably right, but maybe it can avoid peoples pushing wrong things ? I mean, it cost nothing :/

**Comment by NeOMakinG** [comment]
> Are *bundle.js files versionned on purpose?

Yes, in order to deliver the static build to github pages

File: .gitignore (hunk 1 above)
Comment (Progi1984) [reply .gitignore:13]: It must be move to your global gitignore

**Comment by atomiix** [comment]
>Yes, in order to deliver the static build to github pages

I think it can be build automatically before being published to Github pages

**Comment by NeOMakinG** [comment]
I think the deploy failed only because the GH action is not on this repo yet, as it worked on my fork

You can review now @atomiix , I still kicked static files out in order to use a GH action

**Comment by atomiix** [comment]
@NeOMakinG code seems ok but I'm not sure it's just because of the GA missing on the repo. Looks like a rights issue. Also, is it mandatory to keep versioning the assets (even though it's automated now)?

**Comment by NeOMakinG** [comment]
> @NeOMakinG code seems ok but I'm not sure it's just because of the GA missing on the repo. Looks like a rights issue. Also, is it mandatory to keep versioning the assets (even though it's automated now)?

I think we keep assets in order to provide the full project in an npm module easily

The GA is still working on my fork, I'm pretty sure this is nothing

**Review by atomiix — APPROVED** [review]

**Comment by NeOMakinG** [comment]
done @Progi1984

**Review by Progi1984 — COMMENTED** [review]
Normal the file `stories/html/spinner/.index.stories.mdx.swn` or it's a swap file ?

**Comment by NeOMakinG** [comment]
> Normal the file `stories/html/spinner/.index.stories.mdx.swn` or it's a swap file ?

It was a mistake, forgot to add this on my local .gitignore, it's now fine

**Review by Progi1984 — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @NeOMakinG & @sarahdib

**Comment by matks** [comment]
Hi @NeOMakinG when I check Chromatic GitHub Actions I find no successfull runs.
https://github.com/PrestaShop/prestashop-ui-kit/actions/workflows/chromatic.yml
Did the integration of Chromatic work one day or was it an attempt, and we never finished?

**Comment by NeOMakinG** [comment]
> Hi @NeOMakinG when I check Chromatic GitHub Actions I find no successfull runs. https://github.com/PrestaShop/prestashop-ui-kit/actions/workflows/chromatic.yml Did the integration of Chromatic work one day or was it an attempt, and we never finished?

I don't remember but we might did avoid it because of limits

**Pull request merged**